framework/src/modules/chain/rounds/rounds.js

```diff
@@ -214,7 +213,7 @@ class Rounds {
 				.then(() => {
 					if (scope.finishRound) {
 						return promised.land().then(() => {
-							library.bus.message('finishRound', round);
+							library.channel.publish('chain:rounds:change', { number: round });
```

Comment:
Maybe this can be called `chain:rounds:finished` or `finish`?

Reply:
Changing the event key is out of scope of this PR. Please open an issue for that if you feel it's important.